lib/tasks/search.rake

```diff
@@ -19,9 +19,10 @@ namespace :search do
     )
   end
 
-  desc 'Generate new indices, without bringing it online.'
-  task :create_indices => :environment do
```

Comment:
Why did we delete this task?

Reply:
I'm going to move alias addition to the rebuild_indices method.

I removed this command, because we don't add aliases to our indices, but forum search by aliases. As result this behaviour can produce unexpected search results.